src/helm/proxy/clients/anthropic_client.py

```diff
@@ -14,17 +15,228 @@
     TokenizationRequestResult,
     DecodeRequest,
     DecodeRequestResult,
+    TokenizationToken,
 )
 from .client import Client, wrap_request_time, truncate_sequence
+from tokenizers import Tokenizer
+from dataclasses import asdict
+
+
+class AnthropicClient(Client):
+    """
+    Client for the Anthropic models (https://arxiv.org/abs/2204.05862).
+    They use their own tokenizer.
+    """
```

Comment:
Could you add a list of all the Anthropic bugs in the class docstring, and briefly explain how we deal with them?

- Adding the human and assistant prefixes
- Ġ in the tokenizer
- Additional trailing text in completions

There are some docstrings elsewhere about bug workarounds, but it's hard to understand them without the context of what the bugs are.